diff --git a/dist/factories/createPgDriverFactory.js b/dist/factories/createPgDriverFactory.js
index 40cc27c255fa878dc0fb77dc625bbd99599a65d0..47f9099423b9af162e9cfd385e23ada2c53fc1aa 100644
--- a/dist/factories/createPgDriverFactory.js
+++ b/dist/factories/createPgDriverFactory.js
@@ -211,7 +211,8 @@ export const createPgDriverFactory = () => {
                             });
                         }
                         if (Array.isArray(result)) {
-                            throw new InvalidInputError('Must not use multiple statements in a single query.');
+                            // throw new InvalidInputError('Must not use multiple statements in a single query.');
+                            result = result[result.length - 1];
                         }
                         return {
                             command: result.command,
