diff --git a/node_modules/graphql-modules/index.mjs b/node_modules/graphql-modules/index.mjs
index f0ad8ce..98f66aa 100644
--- a/node_modules/graphql-modules/index.mjs
+++ b/node_modules/graphql-modules/index.mjs
@@ -1795,7 +1795,7 @@ function isResolveOptions(value) {
     return isDefined(value.resolve) || isDefined(value.subscribe);
 }
 function isScalarResolver(obj) {
-    return obj instanceof GraphQLScalarType;
+    return obj instanceof GraphQLScalarType || ("name" in obj && ['Percentage', 'TimelessDate'].includes(obj.name));;
 }
 function isEnumResolver(obj) {
     return Object.values(obj).every(isPrimitive);
